lang/de_DE.json

```diff
@@ -0,0 +1,416 @@
+    "Are you sure you want to reset your Connect API key? This will log you out of all emulators.": "Bist du dir sicher, dass du deinen Verbindungs-API-Schlüssel zurücksetzen möchtest? Du wirst beim Zurücksetzen des API-Schlüssel auf allen angemeldeten Emulatoren ausgeloggt.",
+    "Are you sure you want to reset your avatar to the default? This cannot be reversed.": "Bist du dir sicher, dass du dein Profilbild zurücksetzen möchtest? Dieser Schritt kann nicht rückgängig gemacht werden.",
+    "Are you sure you want to reset your web API key? This cannot be reversed.": "Bist du dir sicher, dass du deinen Web-API-Schlüssel zurücksetzen möchtest? Dieser Schritt kann nicht rückgängig gemacht werden.",
+    "Art Team": "Art Team",
+    "Ascending (A - Z)": "Aufsteigend (A - Z)",
+    "Automatically opt in to all game sets": "Automatisch alle Sets eines Spieles aktivieren",
+    "Avatar": "Profilbild",
+    "Beaten": "Beaten",
+    "Beaten (softcore)": "Beaten (softcore)",
+    "Become a Patron": "Werde ein Patreon-Mitglied",
+    "Before or at the time of collecting personal information, we will identify the purposes for which information is being collected.": "Bevor oder während dem Sammeln von persönlichen Daten, werden wir die Gründe für was die Daten eingesammelt wird identifizieren.",
+    "Both": "Beide",
+    "Build your profile, track your progress, and compete with friends on your favorite classic games.": "Erweitere dein Profil, verfolge deinen Fortschritt und messe dich mit Freunden in deinen klassischen Lieblingsspielen.",
+    "By joining the Junior Developer and/or Developer team, you agree to the <1>Developers Code of Conduct</1>.": "Beim Beitreten des Junior Developer und/oder Developer Teams stimmst du dem <1>Verhaltenskodex für Entwickler</1> zu.",
+    "By signing up on RetroAchievements.org, you agree to the <1>Users Code of Conduct</1>.": "Mit der Registrierung auf RetroAchievements.org stimmst du dem <1>Verhaltenskodex für Benutzer</1> zu.",
+    "Cancel Account Deletion Request": "Den Löschungsantrag für deinen Account stornieren",
+    "Cancelled account deletion.": "Die Löschung deines Profils wurde abgebrochen.",
+    "Change Email": "E-Mail ändern",
+    "Change Password": "Passwort ändern",
+    "Changed email address!": "E-Mail-Adresse wurde geändert!",
+    "Changing email address...": "E-Mail-Adresse wird geändert...",
+    "Changing password...": "Passwort wird geändert...",
+    "Changing your email address will revoke your privileges and you will need to have them restored by staff. Are you sure you want to do this?": "Wenn du deine E-Mail-Adresse änderst, so werden deine Privilegien aufgehoben, welche du von dem Admin-Team wiederherstellen musst. Bist du sicher dass du fortfahren möchtest?",
+    "Cheating Reports": "Cheating-Berichte",
+    "Claimed": "Claimed",
```

Comment:
Possible missing translation.

"Beansprucht"?

Reply:
This is pretty hard to say, there is no real translation that I can think of that is fitting the meaning as "Having a set claimed", translating it with "Hat das Set beansprucht" translates more into claiming ownership of something which I am not sure if that sounds accurate

Reply:
That was also my concern which is why I haven't touched the lines for "Claimed"

Reply:
This has been changed to "Reserviert" now